### ISSUE TITLE
Distribute to PyPI

### ISSUE THREAD

**Issue by Lonami** [expectocode/telegram-export#15]
While not an "importable package" (yet?) we could distribute the package as a runnable application to PyPi, so people could run `pip install telegram-export` and then dump chats as easily as `telegram-export --dump whatever`.

In the future, we could add more features that made the package more useful if imported as a Python module. See [`youtube-dl`](https://github.com/rg3/youtube-dl) for example. Most people uses it as a standalone application, but can be imported to further script certain tasks.

**Comment by expectocode** [comment]
definitely should do by 1.0

**Comment by expectocode** [comment]
If we suggest using it as a standalone application, the idea of storing things (config file, output) in the script directory goes out of the window -- we would need to use XDG_CONFIG_DIR or similar and require users to set up an output directory.

**Comment by Lonami** [comment]
There's nothing wrong with that though. See `git` for instance, you need to put a name and email before making a commit, and the configuration can be listed/saved from a TUI directly. Of course, people could still choose to have their own configuration files and use those. Or some arguments tl override less "global" settings like database filename or session name.

**Comment by pielco11** [comment]
I could do a [Docker](https://www.docker.com/) image, that will be updated when a commit is done, so it will be automatically updated, no user interaction. Down side, this size is not so small but you can do more things, it would be not a simple standalone application, but something more.

I'll fully understand that, as the project is now, this could overcome our needs or maybe only me will be using a Docker image. But there are some application that I use that requires docker images, and I'm integrating this tool in that project, so why not give to others the same benefits?

In conclusion, I think that one of the all possible ways to distribute this tool is via Docker.

**Comment by juanvelascogomez** [comment]
In my opinion it would be a great idea to add telegram-export as a pip package. Maybe Docker as @pielco11 said it's another option but the pip one seems easier to develop.

**Comment by expectocode** [comment]
done!
